### PR TITLE
Split up plots

### PR DESCRIPTION
**Pull request by ClFeSc**
~~Based on #32~~

~~Based on #42~~

This PR now contains more changes than just the split-up plots, due to time constraints because of the presentation.

### CODE REVIEW

**Comment by ClFeSc** [comment]
@dkuska @yjojo17 This currently has two potential issues:

1. This currently removes all plots other than the new side-by-side plot. Should it stay this way? Or should I re-enable all/some of the plots?
2. Currently, FPs are not mapped to their missing value result, limiting the usefulness of the plots. Should I try to include this information somehow, or do we assume this is not that important? Also, consider the presentation in two weeks for this.

**Comment by dkuska** [comment]
> @dkuska @yjojo17 This currently has two potential issues:
> 
>     1. This currently removes all plots other than the new side-by-side plot. Should it stay this way? Or should I re-enable all/some of the plots?
> 
>     2. Currently, FPs are not mapped to their missing value result, limiting the usefulness of the plots. Should I try to include this information somehow, or do we assume this is not that important? Also, consider the presentation in two weeks for this.

Let's talk about it on tuesday. However I think reenabling them causes no harm.

About 2.: For the final presentation we need to talk about what we want to present. I feel like mapping to missing-values would be kinda tricky to represent visually.

The TP/FP/FN Plot should showcase the ability of our ML-Model imo.

Additionally we could somewhat represent the distributions of missing-values/sampled-column sizes to show how difficult it is to deduce A [= B from A_s [= B_s.

File: pysrc/utils/descriptive_statistics.py [hunk 1]
```diff
@@ -17,6 +17,10 @@ def file_column_statistics(file_path: str, header: bool, *, delimiter: str =';',
         column_series.sort_values(inplace=True, ascending=True)
         count = column_series.count()
         unique_count = column_series.nunique()
+        # if file_path == '/home/clemens/Dokumente/HPI/WiSe_2022-2023/Approximate_Data_Profiling/GitHub/dkuska/hpi-adp-ind/tmp/COMMENTS__10000_smallest-value_1.csv':
+        #     print(f'{df=}')
+        #     print(f'{unique_count=}, {header=}')
+        #     exit(2)
```
Comment (ClFeSc) [inline pysrc/utils/descriptive_statistics.py]: Note to self: Remove this.

File: pysrc/utils/ind_credibility.py [hunk 2]
```diff
@@ -30,4 +31,9 @@ def ind_credibility(ind: IND, run: 'metanome_run.MetanomeRun', missing_values: i
     if missing_values > baseline_referenced_stat.unique_count - referenced_stat.unique_count:
         # There are more missing values than there are values that are not in the sample of the right hand side
         return nan
-    return (1.0 - missing_values / dependents_stat.unique_count) * run.configuration.credibility()
+    cred = (1.0 - missing_values / dependents_stat.unique_count) * run.configuration.credibility()
+    if cred < 0:
+        eprint(f'Got negative credibility ({cred=}) for {ind=} with {ind.errors=}.\n\t{missing_values=}\n\t{dependents_stat.unique_count=}\n\t{run=}')
+        # eprint(f'{missing_values=}, {dependents_stat.unique_count=}, {run=}, {ind=}')
+        # exit(1)
```
Comment (ClFeSc) [inline pysrc/utils/ind_credibility.py]: Note to self: Remove this.

**Review by ClFeSc — COMMENTED** [review]
@dkuska @yjojo17 I think this is now ready for review.

**Review by dkuska — APPROVED** [review]

**Pull request merged**